lms/djangoapps/discussion_nodebb/migrations/0002_auto_20171017_1002.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please delete these migrations and run makemigrations command to generated single migration.